src/main/java/org/spongepowered/common/mixin/core/tileentity/TileEntityDispenserMixin.java

```diff
@@ -50,4 +56,16 @@
         return new SingleGridLens(0, 3, 3, (Class) TileEntityDispenser.class, slots);
     }
 
+    @Override
+    public void bridge$setDispensedItem(ItemStack dispensed) {
+        this.originalStack = dispensed;
+        this.originalStackSize = dispensed.getCount();
+    }
+
+    @Override
+    public void bridge$restoreDispensedItem(int count) {
+        if (originalStack != null) {
+            this.originalStack.setCount(this.originalStackSize + count);
```

Comment:
What is this restoring exactly? The field seems to exist solely to be stored and updated, but not actually read anywhere that I can see.